src/gate/printer/graphml.h

```diff
@@ -0,0 +1,14 @@
+#include "gate/model/gnet.h"
+
+#include <ostream>
+#include <set>
+#include <string>
+#include <vector>
+
+using GNet = eda::gate::model::GNet;
```

Comment:
Bad practice is to use "using" in headers.

Reply:
Otherwise it is not compiled because definitions are needed in this file

Reply:
Then use the full path in function declarations and remove these usings. The "using" in the header file looks good only inside of class declaration.

Reply:
> Then use the full path in function declarations and remove these usings. The "using" in the header file looks good only inside of class declaration.

!